main.go

```diff
@@ -0,0 +1,167 @@
+package main
```

Comment:
a general comment is that
>  the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.
https://golang.org/doc/effective_go.html#mixed-caps